lib/danger-packwerk/danger_package_todo_yml_changes.rb

```diff
@@ -29,22 +29,26 @@ class DangerPackageTodoYmlChanges < Danger::Plugin
         offenses_formatter: T.nilable(Update::OffensesFormatter),
         before_comment: BeforeComment,
         max_comments: Integer,
-        violation_types: T::Array[String]
+        violation_types: T::Array[String],
+        base_root_directory: String
```

Comment:
Can this accept a `Pathname` instead? With the default being `Pathname.new('.')`

I don't think we need to use the term `base` and `root` since they mean similar things. I think just calling it simply `root_directory` should work!

Also wondering if we should call it `relative_path_to_root`. It doesn't seem like you want to pass in an absolute path here, but rather you want to point the check to a different root relative to where we are now.

Reply:
Yep, definitely on the verbose side here 😁 

`root_path` is what `graphwerk` uses, which was intuitive enough when I just ran into it, so maybe just keeping it as simple as that? https://github.com/samuelgiles/graphwerk/blob/main/lib/graphwerk/builders/graph.rb#L47

Either that or `relative_path_to_root` is fine imo

Reply:
Yep `root_path` is fine too!